pilot/lib/pilot/build_manager.rb

```diff
@@ -241,16 +247,27 @@ def distribute_build(uploaded_build, options)
         if external_group.nil? && options[:groups].nil?
           UI.user_error!("You must specify at least one group using the `:groups` option to distribute externally")
         end
-      else # distribute internally
-        # in case any changes to export_compliance are required
-        if uploaded_build.export_compliance_missing?
-          uploaded_build.save!
-        end
       end
 
       true
     end
 
+    def set_export_compliance_if_needed(uploaded_build, options)
+      build = uploaded_build.find_app_store_connect_build
+      build_attributes = build["attributes"] || {}
+      if build_attributes["usesNonExemptEncryption"].nil?
+        uses_non_exempt_encryption = options[:uses_non_exempt_encryption]
```

Comment:
shouldn't we always set `uses_non_exempt_encryption` if the option `options[:uses_non_exempt_encryption]` provides it? That means the developer wants to override the plist, right?

Reply:
I guess since we use a default, we can't really tell if the plist if being overridden by the developer.

Reply:
@taquitos I currently have `uses_non_exempt_encryption` set to `default_value: false` so I don't want to override it if the plist is set to `true`. I would assume the plist values takes precedence over this. If the plist values is set then `usesNonExemptEncryption` will have a value and we skip it.

So if we always want to set this then maybe I should remove the `default_value: false` and make `optional: true`? But if that is the case then we might run into an issue where plist isn't set and the option isn't set so then we run into this same state 😬

Reply:
Yeah, let's leave it as is. I like your approach.